query.go

```diff
@@ -619,7 +619,7 @@ checkResponses:
 				if curPeer != nil && curPeer.Connected() &&
 					peerTries[curPeer.Addr()] < qo.numRetries {
 
-					curPeer = curPeer
+					curPeer := curPeer
```

Comment:
still don't think this will have any effect, as the pointer `curPer` gets copied into `queryPeer` anyway

Reply:
Belt and suspenders ;)